datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -198,10 +200,10 @@ def inspect_for_gender(
 
 
 def inspect_for_credit_debit_card_number(
-    metadata: Metadata, values: list, config: dict
-) -> tuple:
+    metadata: Metadata, values: List[Any], config: Dict[str, Any]
```

Comment:
what is the type of config here?

Reply:
config is a type of dict (configuration for that particular infotype) 
For eg: 
 _{
        'Prediction_Factors_and_Weights': {
            'Name': 0.4,
            'Description': 0,
            'Datatype': 0,
            'Values': 0.6
        },
        'Name': { 'regex': [<regex patterns>] },
        'Description': { 'regex': [<regex patterns>] },
        'Datatype': { 'type': [<list of datatypes>] },
        'Values': {
            'prediction_type': 'regex/library',
            'regex': [<regex patterns>],
            'library': [<library name>]
        }
    },_

Reply:
Hi @hsheth2 , I had a discussion with @mayurinehate on changing this dict to dataclass, in the future PR we will work on it.